std/range/package.d

```diff
@@ -2442,7 +2442,8 @@ have at most one element.
 
 The type returned by $(D takeOne) is a random-access range with length
 regardless of $(D R)'s capabilities (another feature that distinguishes
-$(D takeOne) from $(D take)).
+$(D takeOne) from $(D take)). One exception: The range can save() only
+if $(D R) can.
```

Comment:
How about using the term `ForwardRange`, e.g.

"However, it is a `ForwardRange` if `range` is forwardable."

(I am not sure whether this makes it clearer)